public/images/ember-core-concepts/ember-core-concepts.svg

```diff
@@ -1,25 +1,25 @@
+<<<<<<< Updated upstream
```

Comment:
@muziejus Thank you for making the new PR! Was on vacation for a few days. 

I think that this `ember-core-concepts.svg` file had a bad rebase or something. Is it possible to perhaps delete the existing file and upload the new one from the external program? (Am not sure what that program is, Adobe Illustrator?)

When looking at it locally, I'm having trouble figuring out what to do in the rebase (line 1, 133, 244, 247, 249 etc). 

BTW if it's easier to discuss on Discord, I'm @ amyrlam on there.

Reply:
@amyrlam do you mean I just send you my files outside of git?

Reply:
(also I'm @muziejus on the ember discord)